nuimo.py

```diff
@@ -307,7 +316,7 @@ def write_matrix(self, matrix, timeout, brightness=1.0):
         led_data = bytearray(matrix_bytes_list)
         led_uuid = self.characteristics_by_name['LED_MATRIX']['uuid']
         led_handle = self.value_handle(led_uuid)
-        self.write_by_handle(led_handle, bytes(led_data))
+        self.write_by_handle_async(led_handle, bytes(led_data), response)
```

Comment:
Thanks a lot for pointing this out, i'll try this way with my app, and thanks for merging :+1: